demo/admin/package.json

```diff
@@ -3,7 +3,7 @@
     "version": "1.0.0",
     "private": true,
     "scripts": {
-        "admin-generator": "rimraf 'src/*/generated' && comet-admin-generator generate crud-generator-config.ts && comet-admin-generator future-generate",
+        "admin-generator": "rimraf 'src/*/generated' && comet-admin-generator generate",
```

Comment:
Can we remove the generate subcommand? Then it would be the same for the API and Admin Generator.

Reply:
You're right, my bad. Must have missed that.

Reply:
It is possible: https://stackoverflow.com/questions/49993576/commander-js-simple-use-case-one-single-file-argument

(we could change it for admin + api generator)

Reply:
I'm open to changing it, but I'm also fine with leaving it like this for now.

Reply:
I would prefer to keep it; it's basically the same API as it was - and maybe we can extend the commands somewhen.